code/modules/projectiles/guns/special/ripper/sawblade.dm

```diff
@@ -279,16 +279,7 @@
 		if (!QDELETED(src))
 			qdel(src)
 
-#undef DROP_DAMAGE
-#undef IMPACT_DAMAGE
-
-#undef EX3_TOTAL
-#undef EX2_TOTAL
-#undef EX1_TOTAL
```

Comment:
these undefs removals need some explanation

Reply:
this was moved to a more appropriate file.

![image](https://user-images.githubusercontent.com/49106866/122157169-60a74f00-ce38-11eb-93a7-0460d07e9f1b.png)